content/docs/atom-guide/look-dev/materials/materials.md

```diff
@@ -0,0 +1,79 @@
+---
+linktitle: Overview
+title: "Material System Overview"
+description: "Materials contain data that control how model surfaces appear in a 3D environment."
+toc: false
+weight: 100
+---
+
+**Materials** contain data that control how model surfaces appear in a 3D environment. All materials have a **material type**, which classifies materials that share the same properties and shader code such as hard surfaces, cloth, or skin. Materials inherit properties either directly from the material type, or from other *parent* materials, and each .material file only needs to store property values that differ from its parent.
+
+<!-- SVG file edited using https://app.diagrams.net/ -->
+![Material Files Diagram](/images/atom-guide/materials/material-file-diagram.svg)
+
+Materials and material types are stored as data items in JSON files. The Atom Material Builder converts the data files into material assets. The material assets are then consumed by the application and applied to a mesh's surface. 
+
+## Materials
+A **material** is a data item that references a material type and defines the material's property values. The material must reference a material type, which defines how the material works and what properties are available. Alternatively, a material might reference another *parent* material and inherit its property values. The material inheritance tree can go as deep as needed. 
+
+Material files (`*.material`) are in JSON format. They can be edited using the Material Editor or authored directly. The file is in a simple JSON format and can be authored or machine-generated by scripting outside the Material Editor.
+
+For more information, see [Material File Specification](/docs/atom-guide/look-dev/materials/material-file-spec/).  
+
+### Material File Example
+
+In the following example, the material uses the **StandardPBR** material type and contains the properties `baseColor`, `roughness`, and `normal`. 
+
+```json
+{
+    "materialType": "Materials/Types/StandardPBR.materialtype",
+    "materialTypeVersion": 6,
+    "propertyValues": {
+        "baseColor.textureMap": "Textures/Default/default_basecolor.tif",
+        "roughness.textureMap": "Textures/Default/default_roughness.tif",
+        "normal.factor": 0.8,
+        "normal.textureMap": "Textures/Default/default_normal.tif"
+    }
+}
+```
```

Comment:
Wondering if there's a `.material` in the source code as bare bones as this? If so, might be better to link to that instead (via github.com/o3de/o3de/tree/) to prevent code samples in docs from getting outdated.

Reply:
IMO it's just as likely that any example .material file in GitHub would change or move in some way that would be detrimental to the documentation.

Reply:
Fair point!